documentation/search-definitions.html

```diff
@@ -82,6 +69,19 @@
 
 
 
+<h2 id="field">field</h2>
```

Comment:
Change to uppercase f in title?

Reply:
I could not make up my mind, as others have lowercase as it is the exact name in the search def. will revisit